docs/pbc/all/tax-management/202404.0/base-shop/third-party-integrations/vertex/install-vertex/integrate-the-vertex-app.md

```diff
@@ -367,35 +367,36 @@ console data:import glossary
+| items[].unitPrice (either GROSS or NET depending on the selected mode)                       | lineItems[].unitPrice                                                    | Prices are converted from Spryker’s cent-based format to Vertex decimal format.                                                                          |
+| items[].merchantStockAddresses.quantityToShip                                                | lineItems[].quantity.value                                               | If `quantityToShip` is less than the quantity requested in cart, the item is mapped to multiple items in the Vertex API.                                 |
+| -                                                                                            | lineItems[].quantity.unitOfMeasure                                       | Always `EA` (“each”). Other units of measure are not supported yet.                                                                                      |
+| items[].taxMetadata                                                                          | Mapped over specific `lineItem` one to one.                              | Metadata is supposed to follow the structure of the Vertex API request. For `lineItems`, it's mapped over each corresponding item based on `lineItemId`. |
+| items[].taxMetadata.seller.company                                                           | lineItems[].seller.company                                               | Required by Vertex from the legal point of view.                                                                                                         |
+| expenses (only for expenses with type `SHIPMENT_EXPENSE_TYPE`)                               | lineItems                                                                | Shipments are treated like products in Vertex - it’s all a line item.                                                                                    |
+| expenses.hash                                                                                | lineItems[].lineItemId                                                   |                                                                                                                                                          |
+| expenses.shipment.shipmentAddress                                                            | lineItems[].customer.destination                                         |                                                                                                                                                          |
+| expenses.shipment.method.shipmentMethodKey                                                   | lineItems[].product.value                                                | Depends on selected shipment method.                                                                                                                     |
```

Comment:
Maybe add in the comment that it is inside '<Product>` tag

Reply:
I didn't get your comment. what's the tag?

Reply:
The Tag is in the logs, when you check request logs in the admin panel the logs are in the cml format, so this value will be in `<Product>Bla</Product>` tag.

Reply:
This doc isn't about vertex logs. Logs in vertax have own standart